test/pageobjects/page.js

```diff
@@ -1,6 +1,22 @@
+import config from "../pageconfig/config.json";
 export default class Page {
     // Global methods go here
-    open (path) {
-        browser.url(path)
+    open(path) {
+        browser.url(path);
+    }
+
+    waitPageToLoad() {
+        browser.waitUntil(
+            function () {
+                const state = browser.execute(function () {
+                    return document.readyState;
+                });
+                return state === "complete";
+            },
+            {
+                timeout: config.TIMEOUT,
```

Comment:
This is nice, adding a config option. Might be neat to take this in as a constructor argument so you could do something like `const page = new Page({timeout: 5000})`

Reply:
agree @m-schrepel. I will modify to make it as part of page constructor.